.github/workflows/build-wheel.yml

```diff
@@ -227,7 +227,7 @@ jobs:
 
       - name: Build cuda.bindings Cython tests
         run: |
-          pip install $(ls ${{ env.CUDA_BINDINGS_ARTIFACTS_DIR }}/*.whl)[test]
+          pip install ${{ env.CUDA_BINDINGS_ARTIFACTS_DIR }}/*.whl --group ./cuda_bindings/pyproject.toml:test
```

Comment:
Do we have to refer to the `pyproject.toml` here? Is there any way we can just use `--group test`?

Reply:
We do, because `pip` only looks for `pyproject.toml` in the passed installation directory, and since we're passing a wheel and not an installation directory it needs to be told where to find the group definition. This is just an unfortunate consequence of `pip install` being a megacommand that does way too many things.